docs/source/checks/tabular/model_evaluation/plot_weak_segments_performance.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+"""
+Weak Segments Performance
+********************************
+
+This notebooks provides an overview for using and understanding the weak segment performance check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+=================================
+The check is designed to help you easily identify the model's weakest segments in the data provided. In addition,
+it is possible to provide a sub list of features for the check thus limiting the search to interesting sub-spaces.
+
+How Deepchecks detects weak segments
```

Comment:
Suggestions:
How Deepchecks Automatically Detects Weak Segments

Or
How Deepchecks Pinpoints Weak Segments

Note - would want it to be clear from the title that we search for them "autonomously"